dataplane/linux/modprobe.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2019 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package intdataplane
+
+import "github.com/projectcalico/felix/iptables"
+
+const (
+	// Modprobe binary on the system
+	cmdModProbe = "modprobe"
+
+	// Kernel module needed for SCTP protocol support on some kernels
+	moduleConntrackSCTP = "nf_conntrack_proto_sctp"
+)
+
+type modProbe struct {
+	module string
+	newCmd iptables.CmdFactory
+}
+
+func newModProbe(module string, newCmd iptables.CmdFactory) modProbe {
```

Comment:
I think the go idiom would be to define a smaller interface here since you're not using most of the methods of the iptables command interface

Reply:
"A little bit of copying is better than a lot of dependency..."

I've defined a new, smaller interface in modprobe.go